Automattic/sensei
# Update language used in email functionality

## Proposed Changes
- Update the description for the student course complete email to "Course Completed". Other email descriptions use past tense, so it doesn't make sense to use present tense for just this email.
- Update the language used for the email recreation tool.

## Testing Instructions


1. In Sensei > Tools, ensure the email creation option looks like the screenshot.
2. Run the tool to recreate the emails.
3. Check that the description for the students course complete email is "Course Completed".

![Screenshot 2023-03-06 at 12 42 05 PM](https://user-images.githubusercontent.com/1190420/223188850-15be6918-3f55-4db2-a006-5ed9b325749b.jpg)

## Pre-Merge Checklist
